docs/05-rfcs/2023-01-11-sdk-spec.md

```diff
@@ -980,18 +1154,24 @@ struct ServiceResponse {
   /**
    * The response's body.
    */
-  body: Serializable;
+  body: Json?;
 }
 ```
 
 ## Table
 
-The table resource represents a database table that can be used to store and query data.
+The table resource represents a NoSQL database table that can be used to store and query data.
```

Comment:
like it! I was confused, but you might want to update `Table (P2) - a relational database table` in the `Planned Resources` section